src/Components/Publishing/ToolTip/Gene.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components"
+import React from "react"
+import { garamond } from "Assets/Fonts"
+import Markdown from "react-markdown"
+import { Truncator } from "../Sections/Truncator"
+import { FollowButton } from "./Components/FollowButton"
+import { NewFeature, NewFeatureContainer } from "./Components/NewFeature"
+
+export interface GeneProps {
+  description?: string
+  href?: string
+  image?: {
+    cropped: {
+      url: string
```

Comment:
Can you link to the query for this? Maybe there are other options available in Gravity that just hasn't been exposed to MP?

Reply:
(for blurriness)

Reply:
I'm not sure what you mean?  There is no query being made atm, assuming that will be covered by the relay work.

Reply:
```{
  gene(id: "graphic-design") {
  	name
    image {
      cropped(width: 240, height: 160) {
        url
      }
    }
    description
    href
  }
}
```

Reply:
Although I'm inserting an exact size, crop seems to be upscaling from a smaller image.

Reply:
Yes, I know there is no query being made, just wanted to see the query you got the mock data from so we can see if there is a non-blurry version we can use instead. The query you provided is what I was looking for -- feel free to task the cropping issue for MP in Jira so we don't forget if you don't want to look into that now.